src/Containers/QRScan.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flex } from '../Utils/Mixins';
+import {
+    Main,
+    MainInterface,
+    Responsive,
+    ResponsiveInterface,
+} from '../Utils/BaseStyles';
+
+export interface QRScanProps
+    extends MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    Icon: React.FC;
+    title: string;
+    InFooter: React.FC;
```

Comment:
Convert this into text and icon separately so you can embedded this into controls.